### DISCUSSION TITLE
Recording caMicroscope contribution on outreachy website

### DISCUSSION THREAD

**Discussion by DIANAAJAMBO** [camicroscope/caMicroscope#800] [Q&A] [answered]
I need assistance with recording a contribution. How can i describe it or what goes under description. Can i for example; post the same bug description that is under github issues on the outreachy website?

**Comment by gbengaoluwadahunsi** [comment, marked as answer]
Hi, just copy the summary of your git pull request and put it there, make sure it's clear and well-detailed.

**Reply by DIANAAJAMBO** [reply]
Thank you @gbengaoluwadahunsi

**Reply by birm** [reply]
I think you answered this question, so I'll add this link which _may_ be helpful too and mark it as an answer. https://www.outreachy.org/apply/make-contributions/

**Comment by gbengaoluwadahunsi** [comment]
Cheers

**Reply by We13b-MD** [reply]
@gbengaoluwadahunsi have you been able to submit final application how did you write your Outreachy internship project timeline section beacuse i guess we need birm for this but i guess the deadline for final application is tomorrow

**Comment by gbengaoluwadahunsi** [comment]
Hello, I haven't submitted it yet.
